Lib/http/server.py

```diff
@@ -1024,6 +1041,10 @@ def _main(args=None):
     parser.add_argument('port', default=8000, type=int, nargs='?',
                         help='bind to this port '
                              '(default: %(default)s)')
+    parser.add_argument('-H', '--header', nargs=2, action='append',
+                        metavar=('HEADER', 'VALUE'),
+                        help='Add a custom response header '
+                             '(can be used multiple times)')
```

Comment:
```suggestion
                             '(can be specified multiple times)')
```

The trace.py argparse uses "specified"

Reply:
Fixed in f0d1bacf24f03419738b4163d150f913ffa5380a